src/js/stories/emptystateview.jsx

```diff
@@ -33,6 +34,14 @@ function contentWithNarrowImage() {
     );
 }
 
+function contentWithIcon() {
```

Comment:
I am not really a fan of multiple component in one file and consider the pattern you show as problematic. Why "hide" a React component in a function, I now it is valid but not really re-usable